src/browser/CameraProxy.js

```diff
@@ -66,7 +66,15 @@ function capture(success, errorCallback) {
         imageData = imageData.replace('data:image/png;base64,', '');
 
         // stop video stream, remove video and button
-        localMediaStream.stop();
+        try{
+            localMediaStream.stop();
+        }catch(e){}
+
+        try{
+            localMediaStream.getVideoTracks()[0].stop();
```

Comment:
you could also get an audio track so this might be better as:

```javascript
localMediaStream.getTracks().forEach(function(mediaStream) {
  mediaStream.stop();
});
```